spec/localhost/sphinx_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+describe 'sphinx installation' do
+
+  describe command('/usr/local/sphinx-2.2.6/bin/indexer') do
```

Comment:
Does serverspec have a way to write this more compact? Versions will change, and keeping them in sync between cookbooks and specs can be a chore.